e2e-app/src/app/tools.po.ts

```diff
@@ -18,7 +18,7 @@ export const sendKey = async(key: string) => {
  * @param message to display in case of error
  */
 export const waitForFocus = async(selector: string, message = `Unable to focus '${selector}'`) => {
-  const {page} = test;
+  const page = getPage();
   const el = await page.$(selector);
   await timeoutMessage(page.waitForFunction(function(_el) { return _el === document.activeElement; }, el), message);
```

Comment:
As we have all the playwright test runner capabilities now, I would use the [toBeFocused ](https://playwright.dev/docs/test-assertions#expectlocatortobefocusedoptions)feature:
```typescript
await expect(selector, message).toBeFocused()
```

Maybe this utility function is not needed anymore, as the code is simple enough to be used inline.